docs/architecture/adr-033-protobuf-inter-module-comm.md

```diff
@@ -19,7 +19,7 @@ service definitions defined in [ADR 021](./adr-021-protobuf-query-encoding.md) a
 
 ## Context
 
-In the current Cosmos SDK documentation on the [Object-Capability Model](../docs/core/ocap.md), it is stated that:
+In the current Cosmos SDK documentation on the [Object-Capability Model](https://github.com/cosmos/cosmos-sdk/blob/master/docs/core/ocap.md), it is stated that:
```

Comment:
```suggestion
In the current Cosmos SDK documentation on the [Object-Capability Model](../core/ocap.md), it is stated that:
```

We should try to avoid master so when this gets added to a versioned doc it doesn't go to master if anything changes.